demo/kc_realm_export.json

```diff
@@ -433,7 +433,7 @@
   }, {
     "id" : "dbcb2359-bfcc-491f-abe7-4fb2e0e5cd33",
     "createdTimestamp" : 1645217917933,
-    "username" : "graphkb_admin",
+    "username" : "pori_admin",
```

Comment:
Should the iprdemo user also be updated in this pr?

Reply:
I think the same. See this comment on users in databases for a quick portrait: https://github.com/bcgsc/pori/issues/57#issuecomment-1154390809.

Reply:
the updated instructions don't work unless this change is made, correct?

Reply:
Will let iprdemo for now as it is still being used by the actual implementation of the demo as per comment https://github.com/bcgsc/pori/issues/57#issuecomment-1244020928